packages/atomic/src/pages/header.js

```diff
@@ -33,22 +33,45 @@ const makeLinks = () => {
     .filter((link) => link.href !== window.location.pathname)
     .map(
       (link) =>
-        `<li style="display: inline-block; list-decoration: none;">
-           <a href="${link.href}" style="margin-right: 10px; color: var(--atomic-primary); text-decoration: none;">${link.label}</a>
+        `<li>
+           <a href="${link.href}">${link.label}</a>
          </li>`
     )
     .join('');
 };
 const example = getCurrentExample();
+const styleTag = document.createElement('style');
+styleTag.innerHTML = `
+  html {
+    nav {
+      padding: 10px 20px;
+      font-family: var(--atomic-font-family);
+      span {
+        font-weight: var(--atomic-font-bold);
+      }
+      ul {
+        display: inline-block;
+        font-size: var(--atomic-text-sm);
+      }
+      li {
+        display: inline-block;
+        list-decoration: none;
+      }
+      a {
+        margin-right: 10px;
+        color: var(--atomic-primary);
+        text-decoration: none;
+      }
+    }`;
+styleTag.setAttribute('nonce', '1234567890');
```

Comment:
`style=` attribute ain't allowed, without `unsafe-inline`. So I moved the style somewhere else, with a nonce.